po/de.po

```diff
@@ -4541,12 +4541,12 @@ msgstr "Schwerer Superschütze"
 #: data/mp/messages/strings/names.txt:777
 #: po/custom/fromJson.txt:1252
 msgid "Super Auto-Cannon Cyborg"
-msgstr "Sturmgeschütz-Supercyborg"
+msgstr "Sturmkanone-Supercyborg"
```

Comment:
I'd add a [Fugen-n](https://de.wikipedia.org/w/index.php?title=Fugenlaut&oldid=186053466) and drop the dash (Sturmkanonensupercyborg) or reverse the order (Supercyborg mit Sturmkanone).

Reply:
> I'd add a [Fugen-n](https://de.wikipedia.org/w/index.php?title=Fugenlaut&oldid=186053466) and drop the dash (Sturmkanonensupercyborg) [...]

Okay.

Reply:
Just a comment: If I have a look at how the original is written shouldn't it be more a "Supersturmkanonencyborg"? Or the original should become an "Auto-Cannon Super Cyborg".

Reply:
True. My understanding is that the bigger cyborgs are "Super Cyborgs". The issue with the German translation (or, German ^^) is that it is not clear what "super" refers to:
* Super-Sturmkanonencyborg: Super version of a "Sturmkanonencyborg"
* Supersturm-Kanonencyborg: Possible, but unlikely to be misunderstood in this way
* Supersturmkanonen-Cyborg: Cyborg with a "Supersturmkanone" (thanks to the dash it might also be misunderstood as one Cyborg with two or more "Supersturmkanonen"

I am not sure to what extent that is a problem in English as well. One way to make things clearer in German is by changing the order, so we get "-supercyborg".

Reply:
> True. My understanding is that the bigger cyborgs are "Super Cyborgs". [...]

Correct. "Super" means "heavy" or "big" and refers to cyborgs' bodies.
A "Heavy Gunner" is a light cyborg, unlike a "Super Heavy-Gunner".

> The issue with the German translation (or, German ^^) is that it is not clear what "super" refers to:
> 
>     * Super-Sturmkanonencyborg: Super version of a "Sturmkanonencyborg"
> 
>     * Supersturm-Kanonencyborg: Possible, but unlikely to be misunderstood in this way
> 
>     * Supersturmkanonen-Cyborg: Cyborg with a "Supersturmkanone" (thanks to the dash it might also be misunderstood as one Cyborg with two or more "Supersturmkanonen"
> 
> 
> I am not sure to what extent that is a problem in English as well. One way to make things clearer in German is by changing the order, so we get "-supercyborg".

In English, the names of all heavy cyborgs start with the word "Super".
Dashes clarify what it refers to if "Cyborg" is no part of these names:

```
msgid "Super Heavy-Gunner"
msgstr "Schwerer Superschütze"

msgid "Super Rail-Gunner"
msgstr "Supermagnetschütze"
```

How about retranslating "Supermagnetschütze" as "Magnet-Superschütze"?

Reply:
> ```
> msgid "Super Heavy-Gunner"
> msgstr "Schwerer Superschütze"
> 
> msgid "Super Rail-Gunner"
> msgstr "Supermagnetschütze"
> ```
> 
> How about retranslating "Supermagnetschütze" as "Magnet-Superschütze"?

Good catch, should definitely be changed for consistency's sake.